### PR TITLE
Remove all feature use, enabling stable rust

### PR DESCRIPTION
**Pull request by cuviper**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -14,8 +14,10 @@ license = "MIT"
 pkg-config = "*"
 
 [dependencies]
+libc = "*"
 log = "*"
 time = "*"
+thread-scoped = "*"
```
Comment (gkoz) [inline Cargo.toml]: I wonder if this is a good time to get rid of wildcard dependecies.

Comment (cuviper) [inline Cargo.toml]: Yeah, probably so.  I guess "x.y" is appropriate? (leaving the third .z unspecified)

Comment (gkoz) [inline Cargo.toml]: I believe "0.x" or "x" (x != 0) should usually suffice if semver is respected.

**Comment by zargony** [comment]
rust-fuse running with stable rustc.. awesome work! Let's update dependency versions and release 0.2.7?

**Comment by cuviper** [comment]
I see you updated the dependencies already.  Yes, I think a new release is warranted. :)

**Pull request merged**